server/routerlicious/packages/protocol-definitions/src/summary.ts

```diff
@@ -53,6 +53,9 @@ export interface ISummaryTree {
 
     // TODO type I can infer from SummaryObject. File mode I may want to directly specify so have symlink+exec access
     tree: { [path: string]: SummaryObject };
+
+    // True if this tree entry is unreferenced. If this flag is not present, the tree is considered referenced.
+    unreferenced?: boolean;
```

Comment:
Looks good, but I wonder if it would be better to have 
```
unreference?: true;
```
I.e. more clearly indicate, that we are not going to ever write unreferenced: false, and that undefined is the way to specify falsey result